djvm-example/src/test/kotlin/com/example/testing/LocalSerialization.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.testing
+
+import net.corda.core.serialization.SerializationContext
+import net.corda.core.serialization.SerializationContext.UseCase.*
+import net.corda.core.serialization.SerializationCustomSerializer
+import net.corda.core.serialization.SerializationWhitelist
+import net.corda.core.serialization.internal.SerializationEnvironment
+import net.corda.core.serialization.internal._contextSerializationEnv
+import net.corda.serialization.internal.*
+import net.corda.serialization.internal.amqp.*
+import org.junit.jupiter.api.extension.AfterEachCallback
+import org.junit.jupiter.api.extension.BeforeEachCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+
+class LocalSerialization : BeforeEachCallback, AfterEachCallback {
```

Comment:
Where is `LocalSerialization` used?

Reply:
It isn't used by this PR and has been "swept up" in all these changes. But it's only a JUnit 5 extension that enables the Corda Serialisation framework, and it _will_ be used soon...